README.md

```diff
@@ -26,16 +26,38 @@ typo3cms phinx:seed:run
 
 Notice that these wrapper commands are executed by TYPO3, thus the full API like `DataHandler` can be used in migrations.
 
+## Migrations
+
 The following paths are used for migrations:
 
 * `typo3conf/ext/*/Migrations/Phinx`
 * `typo3conf/ext/*/Classes/Migrations/Phinx`
 
+Examples to create a migration in a TYPO3 project:
+
+* `vendor/bin/typo3cms phinx:create --path packages/provider/Migrations/Phinx MyMigration`
```

Comment:
```suggestion
* `vendor/bin/typo3cms phinx:create --path packages/provider/Migrations/Phinx MyMigration`
```

Here we should use just `typo3cms` instead to stay in line with the existing docs. It depends on the environment how exactly the TYPO3 Console/CLI is invoked.